apps/challenges/challenge_notification_util.py

```diff
@@ -0,0 +1,36 @@
+import logging
+
+from django.conf import settings
+
+from base.utils import send_email
+from challenges.aws_utils import start_workers
+
+logger = logging.getLogger(__name__)
+
+
+def challenge_start_notifier(sender, instance, field_name, **kwargs):
+    prev = getattr(instance, "_original_{}".format(field_name))
+    curr = getattr(instance, "{}".format(field_name))
+    if prev != curr:
+        challenge = instance
+
+        # Start the challenge worker.
+        response = start_workers([challenge])
+        count, failures = response["count"], response["failures"]
+
+        if(count != 1):
```

Comment:
Why are we using count to check whether the `start_worker` action is successful or not? shouldn't we use a `success` or `failure` flag from response to verify whether action was successful or not?

Reply:
That's how the `start_worker` method is implemented. It returns a dictionary having `count` & `failures`. I can change the methods to return a success flag in another PR since that would require changes in additional files.

Reply:
@KhalidRmb can you add a space after `if`.